packages/api/src/utils/__tests__/sqlObfuscator.test.ts

```diff
@@ -0,0 +1,24 @@
+import * as childProcess from 'child_process';
+
+import { sqlObfuscator } from '../sqlObfuscator';
+
+describe('logParser', () => {
+  it('obfuscates a basic query', async () => {
+    const n = 1;
+    const start = Date.now();
+    for (let i = 0; i < n; i++) {
```

Comment:
nit: probably don't need this loop ?

Reply:
This was for performance testing, I left it in so that if we want to check the timing of N tries it's all set up. Can remove it if it's an issue though 👍